### DISCUSSION TITLE
manta bridge

### DISCUSSION THREAD

**Discussion by mohitbery67** [blockscout/blockscout#9711] [General]
sir/ma'am
I bridged Manta pacific to MANTA ATLANTIC on MANTA DAPP from my metamask(ledger)  wallet adress 0x979Af766340B452335A41991eC9C82183363FC82    to     MANTA wallet adress dfYeHna8b6Uq3zGBiUM9xGVUQTG56YxhTyY8RSdgDH7HPgYFa. the transaction id is  0x9bfae1621ce5d2d0bd833ca0c9af7394af14c80484dfc6fdcc4651b3b4486d04.

the MANTA ATLANTIC tokens are still on the bridge and not yet reached my manta wallet.  kindly help me to recieive the 348.97 manta atlantic token in my manta wallet ( adress given above ) or return back to metamask(ledger)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
